src/operator/tensor/init_op.cc

```diff
@@ -56,7 +70,21 @@ NNVM_REGISTER_OP(zeros_like)
 .add_argument("data", "NDArray-or-Symbol", "The input");
 
 NNVM_REGISTER_OP(ones_like)
-.MXNET_DESCRIBE("Return an array of ones with the same shape and type as the input array.")
+.describe(R"code(Return an array of ones with the same shape and type
+as the input array.
+
+Examples::
+
+  >>> x = mx.nd.zeros((2,3))
```

Comment:
docs in backend are shared for all frontends, so shouldn't have python grammar. See other APIs for example http://mxnet.io/api/python/ndarray.html#mxnet.ndarray.ROIPooling